test-kuttl/e2e/ipfs/05-create-ipfs-cluster.sh

```diff
@@ -12,4 +12,5 @@ spec:
   ipfsStorage: 2Gi
   clusterStorage: 2Gi
   public: false
+	replicas: 2
```

Comment:
it seems like the line spacing is the issue here with the kuttl tests

Reply:
Doh! Thanks.